src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -756,6 +752,24 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
         }
     }
 
+    public class NodeSearchElementVMToBoolConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is NodeSearchElementViewModel)
+                // TODO(Vladimir): Check if bug returns. Earlier line was:
+                //if (value is NodeSearchElement || value is NodeSearchElementViewModel)
```

Comment:
This comment needs to be removed, unless you believe there is anything we need to do here (by creating a new task for it).